pumpkin-macros/src/block.rs

```diff
@@ -0,0 +1,17 @@
+use proc_macro::TokenStream;
+use quote::quote;
+
+pub(crate) fn block_entity_impl(item: TokenStream) -> TokenStream {
+    let input_string = item.to_string();
+    let block_entity_name = input_string.trim_matches('"');
+
+    quote! {
+        pumpkin_world::block::block_registry::BLOCKS
```

Comment:
@Snowiiii I was wondering if this is fine and in another PR, I could do all the changes to bring the methods from block_registry into this macro